fabric-item-api-v1/build.gradle

```diff
@@ -2,5 +2,8 @@ archivesBaseName = "fabric-item-api-v1"
 version = getSubprojectVersion(project, "1.2.1")
 
 moduleDependencies(project, [
-		'fabric-api-base'
+		'fabric-api-base',
+		'fabric-content-registries-v0',
```

Comment:
This dependency should be added in the fabric.mod.json (and the others if they're needed).